envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/permissions/IncomingPermissionsLoggingModeTest.kt

```diff
@@ -691,20 +692,20 @@ internal class IncomingPermissionsLoggingModeTest : EnvoyControlTestConfiguratio
         val echo2Response = callEnvoyIngress(
             envoy = echo2Envoy,
             path = "/log-unlisted-clients",
-            headers = mapOf("x-service-name" to "service-name-from-header"),
-            useSsl = true,
+            headers = mapOf("x-service-name" to "allowed-client"),
```

Comment:
Why are you changing this test and not adding another one?

Reply:
True, I've changed that while developing and forgot about it. I will add another test .a